pyxlsb/styles.py

```diff
@@ -1,9 +1,47 @@
 import os
 from . import recordtypes as rt
 from .recordreader import RecordReader
-
+from .records import FormatRecord
 
 class Styles(object):
+
+    _default_format = {0: {'format': 'General', 'dtype': ''},
```

Comment:
Could you make all of these `FormatRecord` instances? It would offer a better structure than simple dicts and if we move the `dtype` detection logic we no longer need the additional field.

Also, we could rename this to `_builtin_formats` and have `_default_format` be the "General" case which we will return for missing idx's.

Reply:
In terms of aesthetic, could you re-align everything kind of like this, try to keep a low column count:
```python
foo = {
    0: { ... },
    1: { ... },
    ...
}
```